salesforce-oms/1.1/modules/ROOT/pages/order-mgmt-guide.adoc

```diff
@@ -0,0 +1,103 @@
+= SAP S/4HANA Order Management Guide
+:salesforce-oms-version: {page-component-version}
+
+== Overview
+
+Before you implement the https://anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/mulesoft-accelerator-for-salesforce-oms/minor/{salesforce-oms-version}/pages/B2B%20EDI%20Order%20Lifecycle%20Automation/[Salesforce OMS to ERP Integration] use case, you must create a sales order, freight order, outbound delivery, and invoice in SAP.
```

Comment:
```suggestion 
This guide helps you create a sales order, freight order, outbound delivery, and invoice in SAP so that you can implement the https://anypoint.mulesoft.com/exchange/0b4cad67-8f23-4ffe-a87f-ffd10a1f6873/mulesoft-accelerator-for-salesforce-oms/minor/{salesforce-oms-version}/pages/B2B%20EDI%20Order%20Lifecycle%20Automation/[Salesforce OMS to ERP Integration] use case.
```
After reading the whole article, I'd suggest rewording like this.